### ISSUE TITLE
clib install fails: unable to fetch clibs/opencollective:package.json

### ISSUE THREAD

**Issue by vogler** [bpkg/bpkg#110]
~~~
$ clib install bpkg/bpkg
       fetch : bpkg/bpkg:package.json
       fetch : clibs/opencollective:package.json
       error : unable to fetch clibs/opencollective:package.json

$ zsh --version
zsh 5.7.1 (x86_64-apple-darwin18.2.0)

$ clib --version
1.8.1
~~~

**Comment by Potherca** [comment]
We seem to be running into the `package.json` being a shared asset between NodeJS and BPKG.

OpenCollective is a Node dependency, which is why clib can't find it.

As we're not actively using OpenCollective, I'll remove it from the `package.json` and make sure it is not added until we've moved to a non-shared dependecy/package file.

**Comment by Potherca** [comment]
Fix has been released as [`v0.3.1`](https://github.com/bpkg/bpkg/releases/tag/0.3.1)

Thanks for reporting this, @vogler!

**Comment by vogler** [comment]
Thanks for the quick fix!
